python/py-igraph/Portfile

```diff
@@ -78,19 +78,21 @@ if {${name} ne ${subport}} {
     # Testing py312-igraph disabled until all test dependencies get py312 subports.
     if {${subport} ne "py312-${python.rootname}"} {
         # python-igraph optionally makes use of these, and some tests depend on them.
-        # If they are not installed, the corresponding tests will simply be skipped..
+        # If they are not installed, the corresponding tests will simply be skipped.
         depends_test-append     port:py${python.version}-matplotlib \
                                 port:py${python.version}-networkx \
                                 port:py${python.version}-numpy \
                                 port:py${python.version}-pandas \
                                 port:py${python.version}-scipy
+    } else {
+        depends_test-append     port:py${python.version}-numpy
+    }
 
-        pre-test {
-            test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
-        }
-
-        test.run yes
+    pre-test {
+        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
     }
 
+    test.run yes
+
     livecheck.type          none
```

Comment:
this is the default set by the "python PG" and is this redundant.

Reply:
Fixed.